psi4/driver/procrouting/proc_table.py

```diff
@@ -69,6 +69,7 @@
             'dfocc'         : proc.run_dfocc,  # full control over dfocc
             'qchf'          : proc.run_qchf,
             'ccd'           : proc.run_dfocc,
+            'sf-sapt'       : sapt.run_sf_sapt,
```

Comment:
`sf-sapt0`?

Reply:
Well, its not really `sapt0` as you pointed out above. It is currently a bit too hard to integrate everything together without moving SAPT logic more py-side. Py-side SAPT is something that needs to be done for sure, but may be outside of scope for this PR.

Reply:
It's amazing how much functionality you were able to add in so few lines, even if the Py side SAPT is not in place.